egs2/librimix/diar1/local/data.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=100
+FOLDER=git_librimix
+fs=8k
+num_spk="2 3"
+
+ . utils/parse_options.sh || exit 1;
+
+LIBRIMIX=downloads
```

Comment:
this should be referred from `db.sh` instead of a new definition here

Reply:
Added `LIBRIMIX=downloads` to `db.sh` instead of here.